docs/examples/api-reference/data-types.py

```diff
@@ -84,5 +84,5 @@ def test_restrictions(client):
             "timestamp": now,
         },
     ]
-    res = client.log("fennel_webhook", "UserInfoDataset", pd.DataFrame(data))
-    assert res.status_code == 400, res.json()
```

Comment:
Should we still add assertion about the code of the exception being 400?

Reply:
there is no code, pure python exception